addons/mission/functions/fnc_switchAction.sqf

```diff
@@ -1,86 +1,77 @@
 #include "script_component.hpp"
 /*
  * Author: Mike
- * Functionality for toggling switches, When "OFF" switch is red and power levels are 0, when "ON" switch is Green and power levels are either 1 or player defined.
+ * Functionality for toggling switches, when switch is "OFF" the light is red and power levels are 0, when switch is "ON" the light is green and power levels are 1 or player defined.
  *
- * Default state is 0 (OFF) or 1 (ON)
+ * Default state is 0 or 1
  *
- * Can be checked by using getVariable on the object for switch state.
- * My_Switch getVariable ["TAC_Switched", false];
+ * Switch state can be checked by getVariable
+ * My_Switch getVariable ["TAC_switchState", false];
```

Comment:
This should just be `QGVAR(switchState)`. As we now have `MGVAR/QMGVAR` to use them nicely in a mission. _Anyone doing code in the editor deserves pain of writing out `tac_mission_switchState`._